backend/README.md

```diff
@@ -25,7 +25,7 @@ wget https://trusted-setup-halo2kzg.s3.eu-central-1.amazonaws.com/hermez-raw-11
 
 and pass the path to the file to the `Snapshot::new` method.
 
-Furthermore, the `generate_proof_of_solvency` method requires to fetch data about the balances of the wallets of the CEX. This data is fetched using the Covalent API. In order to use this method, you need to create an `.env` file and store the `COVALENT_API_KEY` there. You can get an API key at https://www.covalenthq.com/platform/.
+Furthermore, the `generate_proof_of_solvency` method requires to fetch data about the balances of the wallets of the CEX. the default node url is `http://localhost:8545`.
```

Comment:
This is incorrect. As of now the `generate_proof_of_solvency` method doesn't fetch data at all. We can add here in the readme a description on how to use `fetch_asset_sums` to fetch data from the blockchain. Also we can provide guidance on how to test it (if we set the test to `ignore` by default)

Reply:
Good point!

I will update the README to explain the different methods available for fetching balances. 

There are two primary ways to accomplish this:

You can fetch balances using the `fetch_asset_sums` function.
Alternatively, you can create your own custom fetcher to retrieve the balances.

Once the balances have been obtained, you can proceed to use them in the `generate_proof_of_solvency` function.

Reply:
Updated at [eec59ee](https://github.com/summa-dev/summa-solvency/pull/116/commits/eec59ee96add41075e44f6c9b1c7efa94ffdb361)